app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.startswith("cp"):
+        temp = command.split()
+        old_file, new_file = temp[1], temp[2]
+        if temp[1] != temp[2]:
```

Comment:
You can unpack values from the tuple into variables like this `command, old_file, new_file = command.split()`
And then use these variables without any indexes)
Also, you can use just one if condition with logical operator